docs/user_guide/ttl.md

```diff
@@ -34,7 +58,4 @@ At the moment, there are two ways to perform empty pushes:
 2. Via the Venice Push Job, executed from a Hadoop grid, but with an input directory that contains a file with no 
    records in it.
 
-## Future Work
-A more efficient implementation of TTL is currently under development, which will leverage [Repush](../ops_guide/repush.md)
-in order to do an offline compaction and sorting of the data and thus minimize the impact on servers. This guide will be
-updated when the functionality is ready to try.
\ No newline at end of file
+In order to enable repush with ttl, refer to the [Repush](../ops_guide/repush.md) guide.
```

Comment:
The Usage section should start with: there are two ways to do TTL, via empty push or via TTL Repush. Then have a subsection for each (pulled down here from above).

The numbered list just above this comment should be nested under the Empty Push subsection, IMHO.